lib/sidecar.js

```diff
@@ -987,49 +987,73 @@ class Engine extends ReadyResource {
     await bundle.finalizeTracing()
   }
 
-  async * info ({ key } = {}, client) {
+  async * info ({ key, channel, dir, display } = {}, client) {
```

Comment:
no to `display`, just match flags to keys as we do elsewhere - no nesting

Reply:
removed `display` with nested props